dashboard/README.md

```diff
@@ -1,2 +1,141 @@
+* Even postgres feels like a second class citizen compared to [time series databases.](https://grafana.com/docs/grafana/latest/getting-started/timeseries/)
+* Alternatively, grafana can use our JSON API as a data-source but thats only possible by using a third party plugin. (https://grafana.com/grafana/plugins/simpod-json-datasource)
+* There is no way of adding buttons or formatting inside tables (It may be possible, but I cannot find it) unless we make our own plugins for a customizable table. 
+* Grafana stores layout configuration and all fetched data in its own database (sqlite by default) which is redundant because we store all build data in the packit-service postgresql database as well.
+* Backup and restore will be difficult. (unless we use sqlite as database and copy-paste that sqlite file) (Edit: Incorrect, layout can be exported to json which can be copy -pasted into the correct folder for recreation)
+* Recreating the dashboard from CI will be difficult as layouts are stored in the above mentioned database. (Edit: Incorrect, look at https://github.com/acouvreur/ssh-log-to-influx A standalone grafana instance along with this dashboard can be created via scripts)
+* It has a cli but it can only change settings or passwords and install plugins.
+* We cannot remove the _upgrade to grafana enterprise banner_ in the settings panel.
+* Replacing the grafana logo in the title bar with packit's own is not possible. (unless we edit the source, which is hacky and not ideal)
+* Help menu, login button, settings icon, documentation buttons cannot be removed.
+* Large, publically visible buttons point to grafana documentation, which will be misleading in packit-service's dashboard.
+* We cannot control the URL scheme.
+* I haven't found a way to do something like recursive json.  Eg: (GET and display all builds info and if a user a wants it, find build_id from the all builds json and then do a GET request to fetch another json file for detailed info regarding that build_id)
+* Tests ?
+* Adapting it for our use case will be a literal hackfest with tons of plugins thrown in to make it work.
+
+
+
+## Dashboard - List of Features
+
+### Home Page
+
+* Number of successful and failed builds. (bar chart with dual bars for successful/failed) (toggle for weekly/monthly/yearly)
+* Number of successful out of total builds of all time. (donut chart)
+* Total number of projects (100% filled donut chart)
+* Total number of installations (100% filled donut chart)
+* Manually triggered vs automatic builds. (line chart) (toggle for weekly/monthly/yearly)
+* Top 5 most active projects. (Data List) (toggle for weekly/monthly/all-time)
+* Testing farm usage chart
+
+### Build Info Page
+
+* 
```

Comment:
package name, project, SRPM name, a list of built RPMs, size, link to trigger (PR, release)

Reply:
Yes, sorry. This PR was meant to be work in progress. I'll fill the rest of the pages later today

Reply:
no worries, I was really just suggesting